portugues/colabora/comdiferencas/diff/diff.md

```diff
@@ -1,3 +1,5 @@
 Certo, agora que encontrou as diferencas na versão que introduziu o bug você:
 
+[Corrige as alterações e realiza um "pull request" ao master branch](pull/pull.md)
 
+[Nao sabe corrigir o bug, entao envia ao desenvolvedor as informacoes que conseguiu ate aqui?](../../../revisa/revisar.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.